test/validations.test.js

```diff
@@ -558,7 +559,88 @@ describe('validations', function() {
   });
 
   describe('numericality', function() {
-    it('should validate numericality');
+    it('dummy for should framework which should fail', function(){
+      var t = false;
+      t.should.be.true;
```

Comment:
@Amir-61 now linter should be happy.
Also now using should with the right syntax. But please look how should is used with the other tests in validations.test.js, it's always without the `()`.

```
  describe('commons', function() {

    describe('skipping', function() {

      it('should NOT skip when `if` is fulfilled', function() {
        User.validatesPresenceOf('pendingPeriod', {if: 'createdByAdmin'});
        var user = new User;
        user.createdByAdmin = true;
        user.isValid().should.be.false;
        user.errors.pendingPeriod.should.eql(['can\'t be blank']);
        user.pendingPeriod = 1;
        user.isValid().should.be.true;
      });
```